alcs-frontend/src/app/features/home/gis/gis-subtasks.component.ts

```diff
@@ -15,22 +14,17 @@ import { UserService } from '../../../services/user/user.service';
 })
 export class GisSubtasksComponent implements OnInit {
   subtasks: ApplicationSubtaskWithApplicationDto[] = [];
-  private statuses: CardStatusDto[] = [];
   public gisUsers: UserDto[] = [];
 
   constructor(
     private homeService: HomeService,
     private applicationService: ApplicationService,
     private userService: UserService,
     private router: Router,
-    private applicationSubtaskService: ApplicationSubtaskService
+    private applicationSubtaskService: CardSubtaskService
```

Comment:
since the type changed maybe the name also needs to change: cardSubtaskService?

Reply:
Good catch!